llvm/test/CodeGen/DirectX/llc-pipeline.ll

```diff
@@ -10,10 +10,11 @@
 ; CHECK-NEXT:   DXIL Intrinsic Expansion
 ; CHECK-NEXT:   DXIL Data Scalarization
 ; CHECK-NEXT:   DXIL Array Flattener
+; CHECK-NEXT:   DXIL Resource analysis
```

Comment:
Why did DXIL Resource analysis get moved?

Reply:
It's required by the DXILResourceAccess pass, so it gets scheduled before that pass.

Reply:
This is no longer relevant after splitting the analysis passes